### ISSUE TITLE
la gestion des comptes "auteur" + disparition photo de profil

### ISSUE THREAD

**Issue by spiderwan** [l-atelier-des-chercheurs/dodoc#857]
Il arrive que des photos de profils disparaissent.
L'administrateur qui créé les profils ne peut pas lui même mettre une image de personnalisation, il faut aller dans le profil de la personne concernée. Idéal si l'admin ( si c'est un prof) peut le faire et ainsi réaliser plus rapidement par exemple des profils individuels pour ses élèves ( trombino).

**Comment by louis-ev** [comment]
Ok oui ! 
Est-ce que plus globalement, n'importe qui peut choisir une image de couverture en se créant un compte ? 
J'ajoute un sélecteur "image" sur ce formulaire là ?

<img width="565" alt="image" src="https://github.com/l-atelier-des-chercheurs/dodoc/assets/1948417/6201e421-9228-478d-bffb-e8892e54d2a0">

**Comment by spiderwan** [comment]
yes, selon les utlisateurs, les deux solutions sont interressantes

**Comment by louis-ev** [comment]
En place !
![image](https://github.com/l-atelier-des-chercheurs/dodoc/assets/1948417/80983696-4514-4a2e-991f-e4dca534675d)
